### DISCUSSION TITLE
Image processing support

### DISCUSSION THREAD

**Discussion by myshkin5** [McShelby/hugo-theme-relearn#431] [General]
Is image processing (https://gohugo.io/content-management/image-processing/) supported? Didn't realize Hugo supported this until recently. I see from the Relearn docs (https://mcshelby.github.io/hugo-theme-relearn/cont/markdown/#images) I can do pretty much everything I wanted to. The only nice-to-have I see with Hugo image processing is throwing an error if an image doesn't exist.

Thanks in advance!

**Comment by McShelby** [comment]
Image processing is not supported by the theme and will not in the near future. There are just to many ways to use it. Almost every user will have different requirements how it should work, resulting in a lot of additional configuration flags for the theme.

While there is an render image hook already in place, checking for existence isn't that easy because of the way Hugo manages the file system. This would either result in false negatives or in an increased runtime.

**Comment by myshkin5** [comment]
Good to know. Thanks!
